modules/movies_ui.js

```diff
@@ -0,0 +1,73 @@
+import configureListener from './configure_listeners.js';
+import { handleComment, handleLike } from './event_listeners.js';
+
+import { toJson } from './json_helper.js';
+import LikesService from './likes_service.js';
+// import Movie from './movie';
```

Comment:
- [ ] Required: Kindly remove all unused code from your codebase. It makes your work look more professional.

Reply:
Thank you for your feedback, but those couldn't be optional? 🤔😂
@brytebee

Reply:
I mean all the comments